### PR TITLE
SQL: select all from join table

### PR DESCRIPTION
**Pull request by unrealwork**
## SQL: timezone format
- [x] Added tests for select all from join table

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/sql/select/SqlSelectAllWithJoinTests.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Igor Shmagrinskiy
+ */
+public class SqlSelectAllWithJoinTests extends SqlTest {
+    private static String TEST_PREFIX = "sql-select-all-join-";
```
Comment (oik77) [inline src/test/java/com/axibase/tsd/api/method/sql/select/SqlSelectAllWithJoinTests.java]: final

File: src/test/java/com/axibase/tsd/api/method/sql/select/SqlSelectAllWithJoinTests.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                "sql-select-all-join-metric-2.entity",
+                "sql-select-all-join-metric-1.value",
+                "sql-select-all-join-metric-2.value",
+                "sql-select-all-join-metric-1.datetime",
+                "sql-select-all-join-metric-2.datetime",
+                "sql-select-all-join-metric-1.tags.a",
+                "sql-select-all-join-metric-2.tags.a",
+                "sql-select-all-join-metric-2.tags.b"
+        ));
+        Set<String> resultColumnNames = extractColumnNames(executeQuery(sqlQuery)
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/select/SqlSelectAllWithJoinTests.java]: this part also can be extract to common function

**Comment by oik77** [comment]
I'd better add constants with metric names

**Comment by unrealwork** [comment]
@oik77 I think is not useful for someone who will be read this tests. What we have do when amount of metrics in this test case will grow to few dozen. It will reduce test's readability.

**Comment by oik77** [comment]
@unrealwork if we increase number of metrics, we will need big refactoring

**Comment by oik77** [comment]
LGTM

**Pull request merged**